sycl/test-e2e/Basic/event_profiling_info.cpp

```diff
@@ -11,7 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 // Fails there.
-// UNSUPPORTED: opencl && gpu && gpu-intel-pvc
+// UNSUPPORTED: opencl && architecture-intel_gpu_pvc
```

Comment:
is the name of the LIT varied specified somewhere? this may sound like nitpicking but i would say its
1) too long
2) mix of hyphens and dashes is error prone

if there is no reason we have to use `architecture-intel_gpu_pvc` i would actually prefer we keep `gpu-intel-pvc` or we strip `architecture-` and use `intel_gpu_pvc`

Reply:
Tagging @jzc for awareness/comments as author of #13976

The name is documented in `sycl/test-e2e/README.md`. It is a mix, because I assume that we wanted to have it "namespaced" and therefore it is `prefix-value` where value is what can be passed into `-fsycl-targets` (and there we use hyphens, it is probably easier for parsing there?). Similar to what we have `aspect-value`.

I'm hesitant to change the `value` here, because I think that it is nice that it has 1:1 mapping to what we can pass into `fsycl-targets`. However, I think that we can shorten `architecture` down to `arch`.

Reply:
ok thanks for the explanation, i like the shortening to `arch`. are you planning to do this as part of this PR or in a follow up? thanks

Reply:
> ok thanks for the explanation, i like the shortening to `arch`. are you planning to do this as part of this PR or in a follow up? thanks

I think that we can do this as part of this PR right away since this PR is effectively the first one to use those new LIT features

Reply:
cool thx, i will approve after that change is implemented

Reply:
Should be done in dcb7f03c1cc51e8402b8e9766e2654bc3cad2dc5